#############################
Thu Mar  7 12:50:16 AM EST 2024
--net_train_epochs=50 --net_warmup=10
3788373
In dir: /home/soheil/Sync/umaine/bnn/code/control_lth/output/performance/resnet18/CIFAR10/no_cntr/12345/00-50/
control_epoch: 1
control_iteration:
- 1
control_layer:
- 1
- 2
- 3
- 4
- 5
control_on: 0
control_type: 1
exper_acc_thrd: 95
exper_gpu_id: 1
exper_imp_total_iter: 5
exper_num_trial: 3
exper_prune_perc_per_layer: 0.1
exper_type: performance
net_arch: resnet18
net_batch_size: 64
net_dataset: CIFAR10
net_lr: 0.1
net_pretrained: 1
net_train_epochs: 50
net_train_per_epoch: 2
net_warmup: 10
net_weight_decay: 0.0005

2024-03-07 00:50:18,396 - Using cuda:1 device
2024-03-07 00:50:18,436 - Name of the Cuda Device: NVIDIA RTX A5000
2024-03-07 00:50:18,436 - ####### In performance experiment #######
2024-03-07 00:50:18,437 - In experiment 0 / 3
Files already downloaded and verified
Files already downloaded and verified
2024-03-07 00:50:19,781 - Warming up the pretrained model
2024-03-07 00:50:19,781 - Training...
2024-03-07 00:51:34,446 - Epoch 1 accuracy: 73.8%
2024-03-07 00:53:02,177 - Epoch 2 accuracy: 93.0%
Traceback (most recent call last):
  File "/home/soheil/Sync/umaine/bnn/code/control_lth/src/prune.py", line 791, in <module>
    main()
  File "/home/soheil/Sync/umaine/bnn/code/control_lth/src/prune.py", line 778, in main
    perf_exper(logger, args, device, run_dir)
  File "/home/soheil/Sync/umaine/bnn/code/control_lth/src/prune.py", line 711, in perf_exper
    all_acc, conn, comp = perf_lth(logger, device, args, controller)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/soheil/Sync/umaine/bnn/code/control_lth/src/prune.py", line 493, in perf_lth
    acc, _ = train(model, train_dl, loss_fn, optimizer, None, args.net_warmup, device)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/soheil/Sync/umaine/bnn/code/control_lth/src/network.py", line 222, in train
    running_loss += loss.item()
                    ^^^^^^^^^^^
  File "/home/soheil/.local/lib/python3.11/site-packages/torch/utils/data/_utils/signal_handling.py", line 66, in handler
    _error_if_any_worker_fails()
RuntimeError: DataLoader worker (pid 3790032) is killed by signal: Killed. 
